doc/dev_guide/testing.rst

```diff
@@ -90,47 +90,48 @@ Plot testing
 Plotting is tested using the ``@pytest.mark.mpl_image_compare`` decorator of
 the `pytest mpl plugin <https://pypi.python.org/pypi/pytest-mpl>`_.  This
 decorator uses reference images to compare with the generated output during the
-tests. The references images are located in the folder defined by the argument
+tests. The reference images are located in the folder defined by the argument
 ``baseline_dir`` of the ``@pytest.mark.mpl_image_compare`` decorator.
 
-To run plotting tests, you simply need to add the option ``--mpl``:
+To run plot tests, you simply need to add the option ``--mpl``:
 ::
 
     pytest --mpl
 
-If you don't use the ``--mpl``, the code of the tests will be executed but the
+If you don't use ``--mpl``, the code of the tests will be executed, but the
 images will not be compared to the references images.
 
 If you need to add or change some plots, follow the workflow below:
 
-    1. Write the tests using appropriate decorator such as
+    1. Write the tests using appropriate decorators such as
        ``@pytest.mark.mpl_image_compare``.
-    2. If you need to generate new reference image in the folder
+    2. If you need to generate a new reference image in the folder
        ``plot_test_dir``, for example, run: ``pytest
        --mpl-generate-path=plot_test_dir``
     3. Run again the tests and this time they should pass.
     4. Use ``git add`` to put the new file in the git repository.
 
-When the plotting tests are failling, it is possible to download the figure 
+When the plot tests are failling, it is possible to download the figure 
 comparison images generated by pytest-mpl in the `artifacts tabs 
 <https://ci.appveyor.com/project/hyperspy/hyperspy/build/1.0.2500/job/2c2qccaktd90po2q/artifacts>`_ 
 of the corresponding build.
 
 
-The plotting tests need matplotlib > 3.0.0, since small changes in the way 
+The plot tests need matplotlib > 3.0.0, 
+ since small changes in the way 
  matplotlib generates the figure can make the tests fail.
```

Comment:
There is a space at the beginning of these two line, which set the formatting wrongly. This is not introduced by this PR but it would be good to fix it.

Reply:
Thanks, was not sure if that was intentional or not, but it did look weird ;-)